source/dotnet/Services/WebApi/Controllers/V2/BatchController.cs

```diff
@@ -76,4 +76,17 @@ public async Task<IActionResult> ZipBasisDataAsync([FromBody] Guid batchId)
         var stream = await _basisDataApplicationService.GetZippedBasisDataStreamAsync(batchId).ConfigureAwait(false);
         return Ok(stream);
     }
+
+    /// <summary>
+    /// Get the steps for a specific batch id and grid area code
+    /// </summary>
+    /// <param name="batchId">Batch number</param>
+    /// <param name="gridAreaCode">Grid area code</param>
+    /// <returns>BatchGridAreaStepsDto</returns>
```

Comment:
Do we have such a type? If you do something similar to this, you get a lot more compile-time checking and navigation features:
```suggestion
    /// <returns><see cref="BatchGridAreaStepsDto" /></returns>
```